pytest_fixtures/component/subscription.py

```diff
@@ -2,34 +2,32 @@
 
 
 @pytest.fixture(scope='session')
-def clean_rhsm(default_sat):
+def clean_rhsm(target_sat):
     """removes pre-existing candlepin certs and resets RHSM."""
-    default_sat.remove_katello_ca()
+    target_sat.remove_katello_ca()
 
 
 @pytest.fixture(scope='module')
-def subscribe_satellite(clean_rhsm, default_sat):
+def subscribe_satellite(clean_rhsm, target_sat):
```

Comment:
has to be `module_target_sat`. 

Can't do a straight find-and-replace of all these, unfortunately.